testdata/project-v4-multigroup/config/crd/patches/webhook_in_certmanager_issuers.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: issuers.certmanager.cert-manager.io
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          namespace: system
+          name: webhook-service
+          path: /convert
+      conversionReviewVersions:
+      - v1
```

Comment:
@fischor

See this scaffolds does not seems accurate
Also, I am not sure if the default scaffold is 100% accurate since the path here is fixed to /covert